build_scripts/windows/scripts/az

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+AZ_INSTALLER=MSI "$(dirname "${BASH_SOURCE[0]}")/../python.exe" -IBm azure.cli "$@"
```

Comment:
Maybe we can add some indicator that this is a cygwin/mingw environment?

Reply:
Do you mean add comments or data for telemetry?

Reply:
Perhaps `AZ_INSTALLER=MSI_MINGW`?

Reply:
Telemetry does have shell type information(`Context.Default.AzureCLI.ShellType`), but not accurate now. On windows, it's always `cmd` due to [buggy code](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/telemetry.py#L435-L444). Let's fix it in another PR.

Reply:
Let's not mix shell  information with the package information in AZ_INSTALLER for now. It may not be accurate to assume this script is being called by MinGW. I will investigate how to retrieve the correct shell type in a reliable way.

Reply:
Got it!